zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/internal/client/HttpCall.java

```diff
@@ -158,6 +161,16 @@ public Factory(HttpClient httpClient) {
         ctx -> response.aggregateWithPooledObjects(ctx.eventLoop(), ctx.alloc()),
         // This should never be used in practice since the module runs in an Armeria server.
         response::aggregate);
+    responseFuture = responseFuture.exceptionally(t -> {
+      if (t instanceof UnprocessedRequestException) {
+        Throwable cause = t.getCause();
+        Exceptions.clearTrace(cause);
```

Comment:
What purpose does this serve?  Might be my own personal opinion, but I generally like to see all the causes when possible.

Reply:
I noticed in another change that we don't like to have verbose exception output when it's not really an issue of debugging code (e.g. DNS resolution failure). This is similar, even with a stacktrace for this exception, users wouldn't be able to explore much.

Reply:
makes sense. comment?

Reply:
Sure

Reply:
Works for me :)